README.md

```diff
@@ -1,2 +1,52 @@
-# iroha-python
-Python library for Hyperledger Iroha.
+# Iroha python
+
+Python library for Hyperledger Iroha 2.
+
+## Install
+
+You should install rust (cargo) and python.
+
+```sh
+cd iroha_python
+cargo build --release
+cp target/release/libiroha2_sys.so iroha2/sys/iroha2.so
+python3 test.py
+```
+
+## Generating python sys sources
+
+```sh
+cd iroha_python/generate
+cargo run ../iroha2/sys
```

Comment:
Shouldn't you add args after `--`?
`cargo run -- ../iroha2/sys` maybe?

Reply:
No it is not needed, as `cargo run` never takes arguments without flags